server/routes/departmentForms.js

```diff
@@ -72,6 +83,10 @@ router.get("/form", co.wrap(function* (req, res) {
 router.post("/form", co.wrap(function* (req, res) {
     const form = req.body.form;
 
+    if (!permissionsUtils.isAdmin(req.userDetails)){
```

Comment:
this get should be accessible to all users

Reply:
@may-ben-arie are you sure? isn't it adding the general form?? (Agreed on WhatsApp it's probably a mistake)